Makefile

```diff
@@ -170,6 +170,29 @@ proto_clean_pulsar: ## TODO_TECHDEBT(@bryanchriswhite): Add a proper explanation
 .PHONY: proto_regen
 proto_regen: proto_clean proto_ignite_gen proto_fix_self_import ## Regenerate protobuf artifacts
 
+########################
+### Localnet Helpers ###
+########################
+
+.PHONY: localnet_relayminer1_ping
```

Comment:
Can you move these into a `ping.mk` and add an import at the obbtom of this makefile?